src/express-cli.js

```diff
@@ -44,7 +44,8 @@ import { rpcTest } from '../tests/rpc-tests/rpc-test.js'
 import { constants } from './express/common/constants.js'
 
 import pkg from '../package.json' assert { type: 'json' }
-import { fundGanacheAccounts } from './express/common/ganache-utils.js'
+//import { fundGanacheAccounts } from './express/common/ganache-utils.js'
```

Comment:
Once the PR is ready for final review, please remove all the comments.
Also, note that `fundGanache` account is used in the smoke tests for the CI in bor and heimdall, which need to be updated with the proper anvil setup as well.